Unix/http/http.c

```diff
@@ -1477,7 +1477,11 @@ static MI_Result _CreateSSLContext(Http* self, const char* sslCipherSuite, Serve
     {
         options |= SSL_OP_NO_SSLv3;
     }
-    if ( SSL_CTX_set_options(sslContext, options) == 0 )
+
+    // SSL_CTX_set_options is additive (only adding bits in options). Don't bother with call if options == 0.
+    // Todo: we should error out if the return from options doesn't include at least the options passed in, I would expect
```

Comment:
In wsman_client_auth (the upcoming auth pr)
    if ( options != 0)
    {

        // If options is zero, the operation is a noop. SSL_CTX_set_options only sets, never clears

        MI_Uint32 set_options = SSL_CTX_set_options(sslContext, options);

        // If we don't get the options we asked for, thats an issue.

        if ((set_options & options) != options);
        {
            trace_SSL_CannotSetOptions( set_options ^ options );
            return MI_RESULT_FAILED;
        }
    }